aries-site/src/pages/templates/progressbar.mdx

```diff
@@ -0,0 +1,28 @@
+## Use Cases

```

Comment:
To follow voice and tone, can you sentence case all of these headings? @vavalos5